includes/classes/ToolsPage.php

```diff
@@ -163,4 +164,29 @@ public function get_settings() {
 			],
 		];
 	}
+
+	/**
+	 * Add the action links to the plugin page.
+	 *
+	 * @param array $links The Action links for the plugin.
+	 *
+	 * @return array
+	 */
+	public function filter_plugin_action_links( $links ) {
+
+		if ( ! is_array( $links ) ) {
+			return $links;
+		}
+
+		return array_merge(
+			array(
+				'settings' => sprintf(
+					'<a href="%s"> %s </a>',
+					esc_url( admin_url( 'tools.php?page=block-catalog-tools' ) ),
+					esc_html__( 'Settings', 'block-catalog' )
```

Comment:
I might be wrong here, so feel free to push back, but I'm wondering if this would be best as `Index Blocks` or similar as it's not really a settings page.

Let me know your thoughts.

Reply:
@peterwilsoncc I updated it to `Index Posts` same as the button text on the index page (`/wp-admin/tools.php?page=block-catalog-tools`):
<img width="604" height="498" alt="CleanShot 2025-07-21 at 16 00 07@2x" src="https://github.com/user-attachments/assets/27faae7b-9134-4a48-9636-f5373262be5a" />

Let me know if that makes sense!

Reply:
Index Posts is great, thank you!